roles/hosted_engine_setup/tasks/bootstrap_local_vm/02_create_local_vm.yml

```diff
@@ -63,6 +63,12 @@
           dest: "{{ he_local_vm_dir }}"
           extra_opts: ['--sparse']
       - include_tasks: ../get_local_vm_disk_path.yml
+      - name: Adjust local VM directory permissions
+        file:
+          state: directory
+          path: "{{ he_local_vm_dir }}"
+          recurse: yes
+          mode: 0775
```

Comment:
Why? Also in line 32 above.

Reply:
Using 2770/0770/0660 fails with the following error:
`You will need to grant the 'qemu' user search permissions for the following directories: ['/var/tmp/localvm_ztub84q']`.
Therefore keeping 0755 for sub-directories, 0644 for files in the `localvm` directory, and moving this task right before `Create local VM` task.

Reply:
Set to 0770/0660 and qemu group instead of kvm